requirements.txt

```diff
@@ -41,3 +41,4 @@ pyvo==1.1
 lxml==4.6.3
 suds-py3==1.4.3
 vcrpy==4.1.1
+aiosmtpd==1.4.2
```

Comment:
We have enough dependencies now that it may be worth splitting them out into test and production dependencies.  See, e.g.:

https://github.com/networkx/networkx/blob/main/requirements.txt